### ISSUE TITLE
"suggestion details" in intellisense stopped showing on Ctrl+Space

### ISSUE THREAD

**Issue by munael** [microsoft/pylance-release#4153]
Type: <b>Bug</b>

# Behaviour
## Expected vs. Actual

Pressing Ctrl+Space opens the suggestions menu from intellisense as expected. Pressing it again _should_ show a details panel floating next to the suggestions menu. That panel doesn't show. The preview of the details that should show inline with the suggestion itself when the panel is hidden also do not show.

## Steps to reproduce:

Screenshot:
![image](https://user-images.githubusercontent.com/5769148/227764421-366177aa-2690-40a6-b53f-100d60d6cd29.png)

# Diagnostic data

-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.16
-   Type of virtual environment used (e.g. conda, venv, virtualenv, etc.): Conda
-   Value of the `python.languageServer` setting: Pylance

<details>

<summary>Output for <code>Python</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Python</code>)
</summary>

<p>

```
XXX
```

</p>
</details>

<details>

<summary>User Settings</summary>

<p>

```
Multiroot scenario, following user settings may not apply:

languageServer: "Pylance"

terminal
• activateEnvironment: false

```

</p>
</details>

Extension version: 2023.5.10831011
VS Code version: Code - Insiders 1.77.0-insider (b9226e1ccc11625bcb42b55efb795e07ee533bb0, 2023-03-24T18:57:27.096Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: Yes
Remote OS version: Linux x64 5.4.0-1085-azure

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz (8 x 2112)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.79GB (0.60GB free)|
|Process Argv|--crash-reporter-id 99e317c3-0ab7-4106-9d83-dcf6c481fe98|
|Screen Reader|no|
|VM|0%|

**Comment by debonte** [comment]
Might be a dupe of https://github.com/microsoft/pylance-release/issues/4145?

**Comment by bschnurr** [comment]
CHRF seems to work for me.. it appears to have a squiggle mark under it, so maybe it can't find it.

if you right click 'go to defintion' does it find the class with a class docstring?

I think you might not have the correct python environment selected with `sacrebleu` installed.

**Comment by munael** [comment]
I can navigate to it and intellisense shows me both its signature and a list of members under it.

The same applies to symbols from any other library, including local symbols.

Only the side floating card for details of a highlighted suggestion do not show.

![image](https://user-images.githubusercontent.com/5769148/228160784-95729e79-b46e-491b-9109-be6ef9642ccb.png)

**Comment by bschnurr** [comment]
are you using a notebook?

**Comment by debonte** [comment]
This issue has been fixed in prerelease version 2023.3.41, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023341-29-March-2023-PreRelease)
